app/src/main/java/org/oppia/android/app/player/state/ConfettiConfig.kt

```diff
@@ -0,0 +1,92 @@
+package org.oppia.android.app.player.state
+
+import android.content.Context
+import androidx.core.content.ContextCompat.getColor
+import nl.dionsegijn.konfetti.KonfettiView
+import nl.dionsegijn.konfetti.models.Shape
+import nl.dionsegijn.konfetti.models.Size
+import org.oppia.android.R
+
+/** A sealed class defining the standard configurations for various confetti animations */
```

Comment:
Or perhaps: ``Configuration of initiating a confetti animation in the state player. See sub-classes for available configurations.`` This might be a bit more clear on how the class is meant to be used. It also removes 'sealed class' since that's less important to emphasize here (the developer can just read the class definition to infer that).

Reply:
Done